scripts/caretaking/cherrypick-patch.js

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env node
+
+/**
+ * This script displays the commands that should be run to cherry-pick merged changes
+ * into the current patch branch.
+ *
+ * Run this command like this: `node scripts/caretaking/cherry-pick-patch.js 2019-07-29` where the
+ * date input is the oldest merged pull request that should be patched.
+ *
+ * An authorization token may optionally be provided to increase the API rate limit. A personal
+ * access token string should be added to the command, e.g.:
+ * node scripts/caretaking/cherrypick-patch.js 2019-07-15 4c32652da9e57b2b2IMATOKENLOL8983B3BK
+ */
+
+/** Required argument: Date of the oldest PR to load, should be in the form of YYYY-MM-DD. */
+const date = process.argv[2];
+if (!date) {
+  throw Error('Missing filter date. Provide a date as YYYY-MM-DD, e.g. 2019-06-30');
+}
+
+/** Optional argument: Github personal access token for increased API rate limit. **/
+const authToken = process.argv[3];
```

Comment:
is there a good way to read system environment vars in node? that way we could each export something in our bashrc, i.e. GIT_TOKEN